contracts/kleros/KlerosGovernor.sol

```diff
@@ -559,30 +593,37 @@ contract KlerosGovernor is Arbitrable {
      *  @param _listID The index of the transaction list in the array of lists.
      *  @return txCount The number of transactions in the list.
      */
-    function getNumberOfTransactions(uint _listID) public view returns (uint txCount){
+    function getNumberOfTransactions(uint _listID) public view returns (uint txCount) {
         Submission storage submission = submissions[_listID];
         return submission.txs.length;
     }
 
     /** @dev Gets the number of lists created in contract's lifetime.
      *  @return The number of created lists.
      */
-    function getNumberOfCreatedLists() public view returns (uint){
+    function getNumberOfCreatedLists() public view returns (uint) {
         return submissions.length;
     }
 
     /** @dev Gets the number of the ongoing session.
      *  @return The number of the ongoing session.
      */
-    function getCurrentSessionNumber() public view returns (uint){
+    function getCurrentSessionNumber() public view returns (uint) {
         return sessions.length - 1;
     }
 
     /** @dev Gets the number rounds in ongoing session.
      *  @return The number of rounds in session.
      */
-    function getSessionRoundsNumber(uint _session) public view returns (uint){
+    function getSessionRoundsNumber(uint _session) public view returns (uint) {
         Session storage session = sessions[_session];
         return session.rounds.length;
     }
-}
+
+    /** @dev Gets the cost of the submission.
+     *  @return The cost of the submission.
+     */
+    function getSubmissionCost() public view returns (uint) {
```

Comment:
Adding an extra function which should only be used once is obscuring the code.

Reply:
It is there to avoid reaching the stack limit.